From 4ca6e396f7ea91235fb5b59119c884fab94f57b7 Mon Sep 17 00:00:00 2001
From: Tobias Schramm <tobias@t-sys.eu>
Date: Fri, 8 Nov 2019 22:47:14 +0100
Subject: [PATCH 35/44] arm64: Ignore eDP hpd on the Pinebook Pro

Hotplug detect of the eDP panel is not connected. Thus it needs to be ignored
---
 arch/arm64/boot/dts/rockchip/rk3399-pinebook-pro.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-pinebook-pro.dts b/arch/arm64/boot/dts/rockchip/rk3399-pinebook-pro.dts
index 8140c33ae812..2b4fcebaa692 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-pinebook-pro.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-pinebook-pro.dts
@@ -380,6 +380,7 @@
 	status = "okay";
 	pinctrl-names = "default";
 	pinctrl-0 = <&edp_hpd>;
+	force-hpd;
 
 	ports {
 		edp_out: port@1 {
-- 
2.23.0

